editor/header/index.js

```diff
@@ -59,22 +71,52 @@ function Header( { multiSelectedBlockUids, onRemove, onDeselect } ) {
 			aria-label={ __( 'Editor toolbar' ) }
 			className="editor-header"
 		>
-			<ModeSwitcher />
-			<SavedState />
-			<Tools />
+			<div className="editor-header__left">
```

Comment:
Can we use a more semantic class? Maybe `editor-header__content-tools` and `editor-header__settings` for the right one?